index.html

```diff
@@ -1,35 +1,64 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel='stylesheet' href='https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css'>  
-  <link rel="stylesheet" href="./css/styles.css">
-  <title>Document</title>
-</head>
-<body class="chec">
-  <section class="col-6 mx-auto py-5 mt-5" >
-    <h1 class="text-center" >All Awesome books</h1>
-    
-    <article class="bookListCont">
-    </article>
-  
-    <form action="" id="form" class="flex d-flex flex-column mx-auto align-items-center pt-3 col-8" >
-       <hr class="col-5" > 
-      <h4 class="text-center mx-auto" >Add a new book</h4> 
-      <input type="text" name="" class="w-100 m-3" id="title" placeholder="Title" required > 
-      <input type="text" name="" class="w-100" id="author" placeholder="Author"  required>
-      <input type="submit" value="Add" id="submit" class="p-1 px-2 align-self-end mt-3">
-    </form>
-    
-  </section>
+<html lang="en" class="html">
 
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel='stylesheet' href='https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css'>
+    <link rel="stylesheet" href="./css/styles.css">
+    <title>Document</title>
+  </head>
 
-
-
-
-    <script src='https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js'></script>    
+  <body class="chec">
+    <header class="container-fluid">
+      <nav class="navbar align-self-end">
+        <h1 class="navbar-brand text-capitalize">AnswomeBook</h1>
```

Comment:
- [optional] I think there  is a typo here, you may correct it please (AwesomeBook). 👍